src/components/PhotoCard/PhotoCard.module.scss

```diff
@@ -8,26 +8,20 @@ $black: #000000;
   justify-content: flex-start;
   align-items: flex-start;
   box-sizing: border-box;
-  border-radius: 5px;
-  box-shadow: 0 2px 6px rgba(0,0,0,0.16), 0 2px 6px rgba(0,0,0,0.2);
```

Comment:
I agree with this choice here to remove the box-shadow. I always thought of it as a stand-in. I couldn't really come up with anything that seemed right.

Reply:
I'll have another go.